packages/wonder-blocks-birthday-picker/src/components/birthday-picker.tsx

```diff
@@ -315,7 +336,11 @@ export default class BirthdayPicker extends React.Component<Props, State> {
                     placeholder={this.labels.day}
                     onChange={this.handleDayChange}
                     selectedValue={day}
-                    style={{minWidth: 100, ...dropdownStyle}}
+                    style={{
+                        minWidth: FIELD_MIN_WIDTH_DAY,
+                        ...defaultStyles.input,
+                        ...dropdownStyle,
+                    }}
```

Comment:
**suggestion:** Similar comment about using arrays to avoid object merging conflicts.

```suggestion
                    style={[{
                        minWidth: FIELD_MIN_WIDTH_DAY,
                        },
                        defaultStyles.input,
                        dropdownStyle,
                    ]}
```